handlers.py

```diff
@@ -155,27 +151,26 @@ def split_string_by_keywords(self, string):
 
 
 class WouldPlayHandler(GameExtractionMixin, ContentBasedHandler):
-    fragments = ["I'd play", "id play", "I'd paly", "id paly", "I’d play", "I’d paly", "I’dplay", "I’dpaly", "same to", "same"]
```

Comment:
Why remove these?

Reply:
"I’d play" and "I’d paly" are duplicates